Make ScubaResults.json the default JSON output (#1316)
cisagov/ScubaGear: docs/execution/reports.md

|------------------------------|-----------|
| `IndividualReports`          | This directory contains the detailed reports for each product tested. |
| `BaselineReports.html`       | This HTML file is a summary of the detailed reports. By default, this file is automatically opened in a web browser after running ScubaGear. |
| `ProvideSettingsExport.json` | This JSON file contains all of the information that ScubaGear extracted from the products.  A highly-motivated admin might find this useful for understanding how ScubaGear arrived at its results. Only present if ScubaGear is run with the `KeepIndividualJson` flag; if run without the `KeepIndividualJSON` parameter, the contents of this file will be merged into the ScubaResults.json file. |

> | `ScubaResults.json`           | This JSON file encapsulates all ScubaGear output in a format can could be automatically parsed by a downstream system. It contains metadata about the run and the tenant, summary counts of the test results, the test results, and the raw provider output. Not present if ScubaGear is run with the `KeepIndividualJSON` flag. |

Reviewer: ```suggestion
| `ScubaResults.json`           | This JSON file encapsulates all ScubaGear output in a format that is automatically parsed by a downstream system. It contains metadata about the run and the tenant, summary counts of the test results, the test results, and the raw provider output. Not present if ScubaGear is run with the `KeepIndividualJSON` flag. |
```
